internal/cache/file/cache_handle_test.go

```diff
@@ -877,3 +877,31 @@ func (cht *cacheHandleTest) Test_Read_Random_Parallel_Download_True() {
 	assert.False(cht.T(), cacheHit)
 	assert.ErrorContains(cht.T(), err, util.FallbackToGCSErrMsg)
 }
+
+func (cht *cacheHandleTest) Test_Read_RandomWithNoRandomDownload_And_ParallelDownloadsEnabled() {
```

Comment:
What's "Random download"?

Reply:
it signifies CacheFileforRangeRead flag. renamed the tests.